extensions/amp-sidebar/1.0/amp-sidebar.css

```diff
@@ -10,6 +10,8 @@ amp-sidebar {
   background-color: #efefef;
   height: 100vh;
   top: 0;
+  outline: none;
```

Comment:
FYI setting outline: none breaks a11y if this element needs to receive focus (https://developer.mozilla.org/en-US/docs/Web/CSS/outline#accessibility_concerns).

Reply:
This value was set when the component styling was done initially [here](https://github.com/ampproject/amphtml/commit/7311ed960f9aa71e5160095e3cb64db75fa037f6#diff-ff22105e57037561cfba33a82c88dcf4243939ddcc5318ed895143510eafe913R30-R31)

However, In order to make the sidebar receive focus maybe we should change this in a follow up pr?